scrc/utils/court_names.py

```diff
@@ -6,7 +6,7 @@
 court_issue0 = ["TI_TE", "TI_TPC", "TI_TRAP", "TI_TRPI", "TI_CARP", "TI_CATI"]
 
 # CREATED: # numpy.linalg.LinAlgError: singular matrix (without report)
-court_issue1 = ["AI_XX", "NW_OG"]
+court_issue1 = ["AI_XX", "NW_OG", "CH_WBK", "GL_OG"]
```

Comment:
please use descriptive variable names

Reply:
Initially, I used this list for datasets that got the same error when creating reports to know to exclude the create_report parameter when creating these datasets. 
Now I checked the actual reports and at least most of them are fine, that's why I caught the error and therefore this list isn't needed anymore.

But I used the same naming pattern for specific errors for courts which couldn't be created (`court_error0` e.g.) and I think this makes sense for grouping them like that, especially because these variable names are only used inside the `court_names.py` file.

If you got another idea, let me know.

Reply:
Ok, sounds good. Thanks for the clarification